### ISSUE TITLE
Signing news items

### ISSUE THREAD

**Issue by CJ-Wright** [regro/rever#134]
Should we add something so we can sign news items? This would support CF's UC where the authors are included in the changelog

**Comment by scopatz** [comment]
Sure, PRs welcome!

**Comment by scopatz** [comment]
Don't they just list authors for that release? Not necessarily that category?

**Comment by CJ-Wright** [comment]
Right, but what happens if a single author makes multiple news files? (Sorry by that category I meant the authors catagory).

**Comment by scopatz** [comment]
Oh I wouldn't look at who makes the news files.  I would look at the commit history.  And yes, performing a unique is needed.
